packages/editor/src/components/post-schedule/style.scss

```diff
@@ -9,11 +9,15 @@
 	}
 }
 
-.editor-post-schedule__dialog-toggle {
+.editor-post-schedule__dialog-toggle.components-button {
 	display: block;
 	max-width: 100%;
 	overflow: hidden;
 	text-align: left;
-	text-overflow: ellipsis;
-	white-space: nowrap;
+	white-space: unset;
+	height: auto;
+
+	// The line height + the padding should be the same as the button size.
+	padding: ( ( $button-size - $grid-unit-20 ) / 2 ) 12px;
```

Comment:
Looks like this introduced a deprecation warning when building Gutenberg. Tiny follow-up PR over in https://github.com/WordPress/gutenberg/pull/56412 🙂